redi-connect-front/src/components/Footer.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import "./Footer.scss";
+import twitter from "../assets/images/twitter.svg";
+import medium from "../assets/images/medium.svg";
+import instagram from "../assets/images/instagram.svg";
+import linkedin from "../assets/images/linkedin.svg";
+
+const Footer = () => {
+  return (
+    <div className="footer__wrapper">
+      <div className="row">
+        <div className="footer__container">
+          <div className="footer__container-column footer__container-column-1">
+            <p>ReDi School Website</p>
+            <p>&copy; 2019 By ReDI School</p>
```

Comment:
Can we insert some simple here that uses `Date` to pull the current year and display it here?